hosted/eks/p1/sync_importing_test.go

```diff
@@ -0,0 +1,81 @@
+package p1_test
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+
+	"github.com/rancher/hosted-providers-e2e/hosted/eks/helper"
+	"github.com/rancher/hosted-providers-e2e/hosted/helpers"
+)
+
+var _ = Describe("SyncImport", func() {
+	var cluster *management.Cluster
+
+	When("a cluster is imported for sync", func() {
+
+		BeforeEach(func() {
+			var err error
+			k8sVersion, err = helper.GetK8sVersion(ctx.RancherAdminClient, true)
+			Expect(err).To(BeNil())
+			upgradeToVersion, err = helper.GetK8sVersion(ctx.RancherAdminClient, false)
+			Expect(err).To(BeNil())
```

Comment:
I am not sure of this design. I think it would be better to list available k8s versions for upgrade after the cluster has been imported using `ListEKSAvailableVersions` from rancher/shepherd, wdyt?  Or if you want to continue with this, I would suggest fetching these versions in the BeforeSuite so that we don't keep re-fetching the variable in every block.

Note on `ListEKSAvailableVersions` in eks/helper_cluster.go, I think it is not written properly, it should be using the cluster to fetch available upgrade versions, but currently it simply returns all the available versions. We need to modify it to use `kubernetesversions.ListEKSAvailableVersions` from rancher/shepherd.

Reply:
Raised separate issue for tracking this https://github.com/rancher/hosted-providers-e2e/issues/147